test/integration/01-setup/01-funds-bitcoind-lnds.spec.ts

```diff
@@ -17,6 +17,14 @@ jest.mock("src/phone-provider", () => require("test/mocks/phone-provider"))
 
 const defaultWallet = ""
 
+it("mandatory users", async () => {
+  // load funder wallet before use it
+  await getUserWallet(4)
```

Comment:
These magic numbers are not good. We should not be hard coding IDs.
I think the whole `test_accounts` structure of the `default.yml` could be cleaner but for now perhaps we can simply wrap these in explicit function names:
test/helper/user.ts
```
function getFunderWallet() {
  const testFunderAccountId = 4; # Due to order in default.yml
  return getUserWallet(testFunderAccountId)
}
```

Reply:
agree but I will left this as an exercise for someone to do another pass on test refactoring. don't want to work on this now because it will delay the merge on this PR , and this PR is a dependency for the USD integration.